pkgs/by-name/lu/luau-lsp/package.nix

```
@@ -0,0 +1,53 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  nix-update-script,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "luau-lsp";
+  version = "1.44.1";
+
+  src = fetchFromGitHub {
+    owner = "JohnnyMorganz";
+    repo = "luau-lsp";
+    tag = finalAttrs.version;
+    hash = "sha256-xuFgt0Idl17knZ8fkFFILYq2EixAMP6nefuA0Jx7ISA=";
+    fetchSubmodules = true;
+  };
+
+  NIX_CFLAGS_COMPILE = "-Wno-error";
+
+  nativeBuildInputs = [ cmake ];
+
+  buildPhase = ''
+    runHook preBuild
+
+    cmake --build . --target Luau.LanguageServer.CLI --config Release
+
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    install -D luau-lsp $out/bin/luau-lsp
+
+    runHook postInstall
+  '';
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "Language Server Implementation for Luau";
+    homepage = "https://github.com/JohnnyMorganz/luau-lsp";
+    downloadPage = "https://github.com/JohnnyMorganz/luau-lsp/releases/tag/${finalAttrs.version}";
+    changelog = "https://github.com/JohnnyMorganz/luau-lsp/blob/${finalAttrs.version}/CHANGELOG.md";
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [ anninzy ];
```

Can you also add me as a maintainer, please? :)

```suggestion
    maintainers = with lib.maintainers; [
      anninzy
      HeitorAugustoLN
    ];
```